### PR TITLE
To run js tests with NodeJS, the compiler flags should include `-D nodejs` instead of `-D hxnodejs`

### PR DESCRIPTION
**Pull request by SlavaRa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Features
 
 MUnit has been designed for cross platform Haxe development.
 It currently supports js, swf, as3, neko, c++, java, cs, python and php and the tool chain works on Windows, OSX and Linux.
-To run js tests with NodeJS, the compiler flags should include `-lib hxnodejs` or `-D hxnodejs`.
+To run js tests with NodeJS, the compiler flags should include `-D nodejs`. NOTE: `-lib nodejs`, `-lib hxnodejs` set `nodejs` define automaticaly.
```
Comment (elsassph) [inline README.md]: I guess we can simplify (here and in the cmd help) as:
```
To run js tests with NodeJS, add `-lib hxnodejs` or `-D nodejs`.
```

Comment (SlavaRa) [reply README.md]: moment

Comment (SlavaRa) [reply README.md]: fixed

**Comment by SlavaRa** [comment]
Thanks :)

**Pull request merged**